subsys/net/lib/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ if (CONFIG_DNS_RESOLVER
   add_subdirectory(dns)
 endif()
 
-if(CONFIG_HTTP_PARSER_URL OR CONFIG_HTTP_PARSER OR CONFIG_HTTP_CLIENT)
+if(CONFIG_HTTP_PARSER_URL OR CONFIG_HTTP_PARSER OR CONFIG_HTTP_CLIENT OR CONFIG_HTTP_HPACK_TABLE OR CONFIG_HTTP_SERVER)
```

Comment:
Can we add `CONFIG_HTTP` so this list doesn't keep forever expanding?

Or at least split into multiple lines :)

Reply:
Sure, I will tackle this and many cleanup tasks at some point.